_posts/0000-01-07-mattrt.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```

Comment:
Hey @mattrt maybe the `title` should be changed?

Reply:
interesting, thanks...

@ajaysjain any thoughts here?